src/drivers/nuopc/cime_config/config_component_cesm.xml

```diff
@@ -324,6 +324,7 @@
     <default_value>$ATM_NCPL</default_value>
```

Comment:
Remove the files under src/drivers/nuopc from this PR and create a separate PR to the CMEPS git.